home.html

```diff
@@ -80,7 +80,7 @@
               <img src="images/3.jpg" class="d-block w-100" alt="orange-cup">
               <div class="carousel-caption d-none d-md-block w-50 ml-auto mr-auto">
                 <div class="text_slide1">
-                  <h1>Business Team Mobile</h1>
+                  <h5>Business Team Mobile</h5>
```

Comment:
why? I think h1 is more visible for search engines.

Reply:
But I have a h1 in the home page. This is just a carousel title (temporarily). 
Anyway, I need to add more style in next days to the carousel section and probably I will modify this tags (h1, h5 etc).

Reply:
Ok then!